azure-migrate/VMware migration using ASR/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
All content is already covered by the root `LICENSE` file (which is MIT). Remove.

Reply:
Done